runtime/src/accounts_db.rs

```diff
@@ -7957,11 +7957,14 @@ impl AccountsDb {
         self.bank_hash_stats.lock().unwrap().get(&slot).cloned()
     }
 
+    /// `force_inline_update_index` - prevent farming out to thread pool regardless of the number of accounts updated.
+    ///                               Used by hot loops in BankingStage and ReplayStage.
     fn update_index<'a, T: ReadableAccount + Sync>(
         &self,
         infos: Vec<AccountInfo>,
         accounts: &impl StorableAccounts<'a, T>,
         reclaim: UpsertReclaim,
+        force_inline_update_index: bool,
```

Comment:
I'd rather see this as an enum, if I'm after higher readability. true/false in a calling signature is difficult to read easily.

Reply:
done: 50e4294da3436ecf272a700048f1bde4dcbd058a